e2e/test/scenarios/custom-column/custom-column-reproductions.cy.spec.js

```diff
@@ -1531,3 +1531,52 @@ describe("issue #55940", () => {
       .should("contain", "Offset(Sum([Total]), -1)");
   });
 });
+
```

Comment:
:x: Metric icon has not been fixed, it still shrinks

![image](https://github.com/user-attachments/assets/0a6c0e3c-b696-4446-bd66-47958faa6018)